.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -90,6 +90,7 @@ on:
       registry-username:
         description: "Username to log into the container registry."
         type: string
+        required: false
```

Comment:
Can you update the README?

https://github.com/slsa-framework/slsa-github-generator/blob/13bd59d67f5aeff4317cd943f7cb2bbc8e6d2049/internal/builders/docker/README.md?plain=1#L171